h5c/vic/src/vic-webapp/e2e/app.po.ts

```diff
@@ -277,6 +299,19 @@ export class VicWebappPage {
     }, timeout);
   };
 
+  waitForElementToBePresentXpath(el, timeout = this.opsTimeout, selectBy = 'xpath') {
```

Comment:
This looks like duplicate code of waitForElementToBePresent function. Let's use that one instead if it is.

Reply:
In this case, I have adapted it to a new function waitForElementToBePresentXpath because "waitForElementToBEPresent" is explicit searching by CSS selector, and there are some elements that needs to be found through XPath instead of CSS selectors